packages/@lwc/engine-core/src/framework/base-lightning-element.ts

```diff
@@ -697,11 +696,7 @@ function applyAriaReflectionToLightningElement() {
     applyAriaReflection(LightningElement.prototype);
 }
 
-// The reason for this odd if/else branching is limitations in @lwc/features:
-// https://github.com/salesforce/lwc/blob/master/packages/%40lwc/features/README.md#only-works-with-if-statements
-if (features.DISABLE_ARIA_REFLECTION_POLYFILL) {
-    applyAriaReflectionToLightningElement();
-} else if (!process.env.IS_BROWSER) {
+if (!process.env.IS_BROWSER || lwcRuntimeFlags.DISABLE_ARIA_REFLECTION_POLYFILL) {
```

Comment:
This is an example of simplifications we can now do, since the restrictions are lifted